lib/midcom/helper/datamanager2/widget/tinymce.php

```diff
@@ -284,10 +285,50 @@ private function _get_imagetools_configuration()
     
         return <<<EOT
 imagetools_toolbar: "editimage imageoptions",
-images_upload_url: "/__ais/imagepopup/upload/image/{$guid}/",
-images_upload_base_path: "/",
-images_upload_credentials: true,
 imagetools_cors_hosts: ['{$hostname}'],
+setup: function(editor) 
```

Comment:
Can you move these functions into a JS file and then just reference them here? Would make it more maintainable